Source/WebCore/html/FeaturePolicy.cpp

```diff
@@ -77,15 +77,21 @@ static const char* policyTypeName(FeaturePolicy::Type type)
     case FeaturePolicy::Type::XRSpatialTracking:
         return "XRSpatialTracking";
 #endif
+    case FeaturePolicy::Type::PrivateToken:
+        return "PrivateToken";
     }
     ASSERT_NOT_REACHED();
     return "";
 }
 
-bool isFeaturePolicyAllowedByDocumentAndAllOwners(FeaturePolicy::Type type, const Document& document, LogFeaturePolicyFailure logFailure)
+bool isFeaturePolicyAllowedByDocumentAndAllOwners(FeaturePolicy::Type type, const Document& document, LogFeaturePolicyFailure logFailure, std::optional<SecurityOriginData> securityOrigin)
```

Comment:
I am not exactly sure why we need a securityOrigin here, the one of the document and its ancestors should be sufficient in theory.